tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -96,7 +96,15 @@ TfLiteStatus MicroInterpreter::PrepareNodeAndRegistrationDataFromFlatbuffer() {
     auto* opcodes = model_->operator_codes();
     BuiltinDataAllocator* builtin_data_allocator =
         allocator_.GetBuiltinDataAllocator();
-    for (size_t i = 0; i < subgraph->operators()->size(); ++i) {
+
+    // TODO(b/192589496): Ops must always be there. Remove this check when fixed
+    int operators_size;
```

Comment:
Maybe even this can call the helper method, if it's a public method in MicroGraph.